config/notifiers.go

```diff
@@ -870,6 +882,77 @@ func (c *JiraConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	if c.IssueType == "" {
 		return fmt.Errorf("missing issue_type in jira_config")
 	}
+	return nil
+}
+
+type RocketchatAttachmentField struct {
+	Short *bool  `json:"short"`
+	Title string `json:"title,omitempty"`
+	Value string `json:"value,omitempty"`
+}
+
+type RocketchatAttachmentActionType string
+
+type RocketchatMessageProcessingType string
+
+const (
+	ProcessingTypeSendMessage        RocketchatMessageProcessingType = "sendMessage"
+	ProcessingTypeRespondWithMessage RocketchatMessageProcessingType = "respondWithMessage"
+)
+
+type RocketchatAttachmentAction struct {
+	Type               RocketchatAttachmentActionType  `json:"type,omitempty"`
```

Comment:
I feel this type is unnecessary since we don't do anything with it. Can we just change it to `string`?

```suggestion
	Type               string  `json:"type,omitempty"`
```